packages/browser/test/unit/sdk.test.ts

```diff
@@ -127,4 +128,63 @@ describe('init', () => {
     expect(newIntegration.setupOnce as jest.Mock).toHaveBeenCalledTimes(1);
     expect(DEFAULT_INTEGRATIONS[1].setupOnce as jest.Mock).toHaveBeenCalledTimes(0);
   });
+
+  describe('initialization error in browser extension', () => {
+    const DEFAULT_INTEGRATIONS: Integration[] = [
+      new MockIntegration('MockIntegration 0.1'),
+      new MockIntegration('MockIntegration 0.2'),
+    ];
+
+    const options = getDefaultBrowserOptions({ dsn: PUBLIC_DSN, defaultIntegrations: DEFAULT_INTEGRATIONS });
+
+    it('should not log a browser extension error if executed inside regular browser environment', () => {
+      const consoleErrorSpy = jest.spyOn(console, 'error').mockImplementation(() => {});
+
+      init(options);
+
+      expect(consoleErrorSpy).toBeCalledTimes(0);
+
+      consoleErrorSpy.mockRestore();
+    });
+
+    it('should log a browser extension error if executed inside a Chrome extension', () => {
+      const consoleErrorSpy = jest.spyOn(console, 'error').mockImplementation(() => {});
+
+      Object.defineProperty(WINDOW, 'chrome', {
```

Comment:
I would add an `afterEach` block that resets this back to undefined, to ensure we don't leak this into other tests!

Reply:
I added this block to reset it and I also added the test for the regular browser environment at last to make sure this fails directly if `browser` or `chrome` is still part of the globals.